console2/src/components/organisms/ProcessActivity/index.tsx

```diff
@@ -131,7 +131,15 @@ const ProcessActivity = (props: ExternalProps) => {
         setProcess(process);
 
         if (process.parentInstanceId && !rootProcessRef.current) {
-            rootProcessRef.current = await apiGetRoot(process.parentInstanceId);
+            apiGetRoot(process.parentInstanceId)
+                .catch(() => {
+                    rootProcessRef.current = null;
+                })
+                .then((rootProc) => {
```

Comment:
Why not use `await` with try-catch? The outer function is already async.

Reply:
Fair. reverted to the simpler approach. Going in i thought I might have to account for more outcomes. Sometimes it's a 204 no-content response, others returns 400 bad request depending on parent-child(-child...) depth and what does or doesn't exist.